src/components/AdminPanel/AdminMedCenters/MedCenters/MedCentersList/MedCentersList.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { fetchMedCenters, medCenterDelete } from '@store/actionCreators/medCenters';
+import { useTypesSelector } from '@hooks/UseTypedSelector';
+import Button from '@components/common/Button/Button';
+import MedCenter from '@components/MedCentersPage/MedCentersList/MedCenter/MedCenter';
+import { MedCentersListWrapper, CenterWrapper } from './MedCentersListStyle';
+
+interface IMedCentersListProps {
+  searchValue: string;
+}
+
+const MedCentersList: React.FC<IMedCentersListProps> = ({ searchValue }) => {
+  const dispatch = useDispatch();
+  const { medCenters } = useTypesSelector((state) => state.medCenter);
+
+  const deleteMedCenter = (idMedCenter: string) => {
+    dispatch(medCenterDelete(idMedCenter));
+  };
+  
+  useEffect(() => {
+    dispatch(fetchMedCenters());
+  }, []);
+
+  const filtrationСondition = (medCenter: { name: string }) =>
+    medCenter.name.toLowerCase().includes(searchValue.toLowerCase());
+
+  return (
+    <MedCentersListWrapper>
+      {medCenters
+        .filter((medCenter) => filtrationСondition(medCenter))
```

Comment:
what for calling here one more function if the whole filter can be just a separate variable?

Reply:
that is .filter(variable)

Reply:
@KLIM-nk01 nope
```
const filteredArr = arr.filter(elem => condition);

//render
filteredArr.map(elem => <Component />);
```

Reply:
fixed